.github/workflows/pr.yml

```diff
@@ -373,6 +373,7 @@ jobs:
           path: designer/client/junit.xml
 
   cypressTests:
+    if: false
```

Comment:
Please remember to remove it before merge

Reply:
removed it :+1: